### PR TITLE
getting environment from app.settings.env

### PR DESCRIPTION
**Pull request by vshushkov**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,7 +28,16 @@ module.exports = function setupTestFixtures(app, options) {
     fixturesPath: '/server/test-fixtures/'
   }, options);
 
-  if (options.environments.indexOf(process.env.NODE_ENV) === -1) return;
+  var environment = app.settings && app.settings.env
+    ? app.settings.env : process.env.NODE_ENV;
+
+  var match = Array.isArray(options.environments)
+    ? include.indexOf(environment) !== -1
```
Comment (JonathanPrince) [inline index.js]: where does 'include' come from?

**Comment by JonathanPrince** [comment]
Thank you @vshushkov for you work on this PR, especially for fixing the breaking test :+1: 

The environment matching is a real improvement. Is the include in `include.indexOf(environment)` a mistake or is there something i'm missing?

**Comment by vshushkov** [comment]
Hi, @JonathanPrince 
Yep, it was a mistake) Fix it and cover by test.

**Comment by JonathanPrince** [comment]
Hi @vshushkov, thank you for contributing!

**Pull request merged**